auth-api/migrations/versions/2025_02_27_2ef3f0be3759_26014_permissions_updates.py

```diff
@@ -0,0 +1,76 @@
+"""26014-permissions-updates
```

Comment:
Will rebase after @rodrigo-barraza's PR for membership invitation flow is moved in.
1) Update down_revision
2) Double check and re-sync permissions for external staff once UI clean up complete